community-content/tf_agents_bandits_movie_recommendation_with_kfp_and_vertex_sdk/mlops_pipeline_tf_agents_bandits_movie_recommendation/mlops_pipeline_tf_agents_bandits_movie_recommendation.ipynb

```diff
@@ -958,156 +900,146 @@
       },
       "outputs": [],
       "source": [
-        "@dsl.pipeline(\n",
-        "    pipeline_root=PIPELINE_ROOT,\n",
-        "    name=f\"{PIPELINE_NAME}-startup\")\n",
+        "from kfp.components import load_component_from_url\n",
+        "\n",
+        "generate_op = load_component_from_url(\n",
```

Comment:
Could you replace these urls with the official ones after it's merged? For 
https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/community-content/tf_agents_bandits_movie_recommendation_with_kfp_and_vertex_sdk/mlops_pipeline_tf_agents_bandits_movie_recommendation/src/generator/component.yaml

Reply:
I can do that even without merging =)
Done.